app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.players.player import Player
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    total_rating = 0
+    for player in players:
+        total_rating += player.get_rating()
+    return total_rating
+
+
+def elves_concert(elfs: list[Elf]) -> None:
```

Comment:
Consider renaming the parameter `elfs` to `elves` to follow the correct plural form of 'elf'. This will improve code readability and consistency.